pkg/export/service.go

```diff
@@ -162,11 +162,16 @@ func (e *exporter) mqttConnect(conf config.Config, logger logger.Logger) (mqtt.C
 	opts := mqtt.NewClientOptions().
 		AddBroker(conf.MQTT.Host).
 		SetClientID(e.id).
-		SetCleanSession(true).
+		SetCleanSession(false).
```

Comment:
I think this should be optional, so that when started for the first time it can start with the new session. Should be added to ENV vars.

Reply:
added variable to the env in new commit